examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<blockquote>the DM has one additional parameter that controls this</blockquote><p>It's not strickly one additional parameter right? It uses a different (random) parameter. Maybe something like: unlike the Multinomial where the probability vector is fixed, the DM allows for a slightly different probability vector for each batch of observations, which is drawn from a common Dirichlet distribution.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<blockquote>It's not strickly one additional parameter right? It uses a different (random) parameter.</blockquote><p> </p><p>Hmm...I'm not sure that distinction makes sense to me. The multinomial is parameterized by p on the k-simplex (k-1 dimensions), and the DM is parameterized by alpha in the positive reals (k dimensions). Am I using "parameter" differently?</p><p> </p><blockquote>unlike the Multinomial where the probability vector is fixed, the DM allows for a slightly different probability vector for each batch of observations, which is drawn from a common Dirichlet distribution.</blockquote><p> </p><p>I'm fine with using this intuitive explanation for what the DM is, but it may be redundant with some of the explanation that's currently below, closer to the simulation (i.e. "Each observation from the DM is simulated by..."). I can imagine an argument for including this information near the top, though.</p><p> </p>

Reply:
Yeah I agree you say it lower. I would vote to have it only at the top

Reply:
Cool, I think I addressed most of your comment on this text.

> It's not strickly one additional parameter right? It uses a different (random) parameter.

Was there something else you're getting at here? Maybe related to @AlexAndorra's suggestion that I better explain my `alpha = frac * conc` parameterization?

Reply:
<p>I still think the one extra parameter is misleading. While it's true that the Dirichlet has k free parameters and the DM has k-1 (because we normalize them), this is not where it's flexibility is coming from. You can model the Multinomial with a single Dirichlet prior on the p and it still won't be as flexible as the DM. It's the fact that each batch of observations is allowed a different draw from this Dirichlet that makes it more flexible.</p>

Reply:
<p>Also the concentration is not a parameter of the general DM (it is of your particular data generating model below). I wouldn't mention it yet. Specially because it sounds like this is the"extra" parameter that distinguishes the DM from the Multinomial.</p>

Reply:
<p>Yeah, I agree with Ricardo here: first introducing the DM as a generalization of Beta-Binomial (as you already started doing, this is just a small tweak, and as Ricardo does here: "It's the fact that each batch of observations is allowed a different draw from this Dirichlet that makes the DM more flexible than the Multinomial").</p><p>And then, later in the NB, better explaining your <code>alpha = frac * conc</code> parameterization -- when you actually use it I think is a good place</p>

Reply:
<p>I've updated this text (and the explanation of the simulation below). Does the current iteration address your concerns about the reparameterization and the explanation for the DM's flexibility?</p>

Reply:
<p>I think the new text is much better. Now I would only cut back a bit on redundancy: </p><p> </p><blockquote>Unlike the <a href="https://en.wikipedia.org/wiki/Multinomial_distribution" target="_blank">Multinomial distribution</a>, in which the variance of counts is entirely  determined by the sample size and the probability of each category,  the DM is more flexible than the multinomial, allowing it to  model counts that are <em>more variable</em> than the Multinomial  would predict.  The Dirichlet-multinomial can be understood as draws from the multinomial  but where each sample has a slightly different probability vector,  which is itself drawn from a common <a href="https://en.wikipedia.org/wiki/Dirichlet_distribution" target="_blank">Dirichlet distribution</a>. </blockquote><p>I think you don't need to mention the sample size parameter, as this one works exactly the same in the M and DM. I think something shorter like this might do the job better:</p><p> </p><blockquote>The Dirichlet-multinomial can be understood as draws from a Multinomial distribution where each sample has a slightly different probability vector, which is itself drawn from a common <a href="https://en.wikipedia.org/wiki/Dirichlet_distribution" target="_blank">Dirichlet distribution</a>. This contrasts with the Multinomial distribution, which assumes that all observations arise from a single fixed probability vector. This enables the Dirichlet-multinomial to accommodate more variation in count data than the Multinomial. </blockquote>

Reply:
<p>Yep, all good now, thanks! I also like Ricardo's proposal -- it's up to you</p>